From: Tanzir Hasan <tanzirh@google.com>
Subject: lib/trace_readwrite.c:: replace asm-generic/io with linux/io
Date: Thu, 21 Dec 2023 20:32:33 +0000

asm-generic/io.h can be replaced with linux/io.h and the file will still
build correctly.  It is an asm-generic file which should be avoided if
possible.

Link: https://lkml.kernel.org/r/20231221-tracereadwrite-v1-1-a434f25180c7@google.com
Signed-off-by: Tanzir Hasan <tanzirh@google.com>
Suggested-by: Al Viro <viro@zeniv.linux.org.uk>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/trace_readwrite.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/trace_readwrite.c~lib-replaced-asm-generic-io-with-linux-io
+++ a/lib/trace_readwrite.c
@@ -7,7 +7,7 @@
 
 #include <linux/ftrace.h>
 #include <linux/module.h>
-#include <asm-generic/io.h>
+#include <linux/io.h>
 
 #define CREATE_TRACE_POINTS
 #include <trace/events/rwmmio.h>
_
